ui/admin/app/templates/scopes/scope/targets/target/add-egress-worker-filter.hbs

```diff
@@ -0,0 +1,35 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+}}
+
+{{page-title (t 'resources.target.workers.actions.add-egress-worker-filter')}}
+<Breadcrumbs::Item
+  @text={{t 'resources.target.workers.actions.add-egress-worker-filter'}}
+  @route='scopes.scope.targets.target.add-egress-worker-filter'
+/>
+
+<Rose::Layout::Page as |page|>
+
+  <page.breadcrumbs>
+    <Breadcrumbs::Container />
+  </page.breadcrumbs>
+  <page.header>
+    <h2>
+      {{t 'resources.target.workers.actions.add-egress-worker-filter'}}
+      <DocLink @doc='target.worker-filters' @iconSize='large' />
+    </h2>
+    <p>{{t 'resources.target.workers.ingress-workers.description'}}</p>
```

Comment:
Should this be `egress-workers` in the translation?

Reply:
ohh yes great catch!! thank u